server/proxy/server.js

```diff
@@ -93,7 +93,7 @@ app.get('/createRoom', (req, res) => {
         jar: cookieJar,
         withCredentials: true
     });
-    axiosInstance.post('api/room/create')
+    axiosInstance.post('api/room/create', { number: 3 })
```

Comment:
`number` isn't very descriptive, could you rename it to something like `numRounds` or `rounds`?